pubspec.yaml

```diff
@@ -80,6 +80,7 @@ dependencies:
   stack_trace: ^1.11.0
   firebase_crashlytics: ^3.3.7
   geolocator: ^10.1.0
+  adhan: ^2.0.0+1
```

Comment:
kak mau tanya dengan menggunakan library ini apakah berarti kita tidak perlu hit ke API get Prayer Times By Longitude Latitude yang sudah dibuat Oleh tim BE ?

Link API : https://docs.google.com/document/d/1R_3YCr2BzPUg7jb3JUv3DcKY8rLxao_o_ogGWSFJZ3Q/edit#heading=h.mkib1qbpglu2

Reply:
nggak perlu rhak, hrsnya lebih hemat resource sih jdnya gak perluu pakai API